lib/tokenlist.cpp

```diff
@@ -267,12 +267,14 @@ Token *TokenList::copyTokens(Token *dest, const Token *first, const Token *last,
     std::stack<Token *> links;
     Token *tok2 = dest;
     int linenr = dest->linenr();
```

Comment:
this looks inconsistent. we read linenr from dest but column from dest->next().
should there be some more option for the location?